dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.cdi3.0-websocket2.0.feature

```diff
@@ -0,0 +1,12 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.cdi3.0-websocket2.0.feature
+visibility=private
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-Provision-Capability: \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.websocket-2.0)))", \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.cdi-3.0)))"
+IBM-Install-Policy: when-satisfied
+-bundles=com.ibm.ws.wsoc.cdi.weld.jakarta
+kind=noship
```

Comment:
Can you open an issue to get this put into beta once we get it delivered and into some builds? Assign it to me and I'll take care of getting it completed.

Reply:
Also in that same issue we should note looking at adding `WLP-Activation-Type: parallel`

Reply:
Ah, that's right.